Makefile

```diff
@@ -33,27 +29,26 @@ FAIL_COVERAGE = { echo '$(COLOUR_RED)FAIL - Coverage below $(COVERAGE)%$(COLOUR_
 .PHONY: test cover showcover
 
 # --- Lint ---------------------------------------------------------------------
-
 GOLINT_VERSION = 1.27.0
 GOLINT_INSTALLED_VERSION = $(or $(word 4,$(shell golangci-lint --version 2>/dev/null)),0.0.0)
 GOLINT_MIN_VERSION = $(shell printf '%s\n' $(GOLINT_VERSION) $(GOLINT_INSTALLED_VERSION) | sort -V | head -n 1)
 GOPATH1 = $(firstword $(subst :, ,$(GOPATH)))
-LINT_TARGET = lint-$(if $(filter $(GOLINT_MIN_VERSION),$(GOLINT_VERSION)),local,with-docker)
+LINT_TARGET = $(if $(filter $(GOLINT_MIN_VERSION),$(GOLINT_VERSION)),lint-with-local,lint-with-docker)
 
```

Comment:
Consider making it just LINT ?

Reply:
As per usual make conventions, I would expect `$(LINT)` to be the lint program itself, much like `$(CC)` is the c compiler, `$(YACC)` is yacc, etc. I think I prefer to keep this as `LINT_TARGET`.